### DISCUSSION TITLE
FairnessDashboard on a specific port

### DISCUSSION THREAD

**Discussion by DevLugo** [fairlearn/fairlearn#724] [Q&A] [answered]
Hi Guys.

I'm working on creating a service that instantiates the FairnessDashboard on a random port for each request.

I was trying to add the port when I create the dashboard like this:

`FairnessDashboard(sensitive_features=sensitive_features_test, y_true=Y_test, y_pred=y_pred, port=8080)`

but this is retrieving a "port nor available" error, I already tried with different random port and don't work :S 

Any suggestion?

**Comment by romanlutz** [comment, marked as answer]
Hi @DevLugo !
As I just wrote on #725 this is a `raiwidgets` question rather than a `fairlearn` question. Please open issues in the [corresponding repo](https://github.com/microsoft/responsible-ai-widgets). It would be great to understand what you're trying to do. Perhaps you can elaborate in the new issue you create in that repo. FWIW these dashboards choose a port themselves if you don't provide one. It would also be useful to know where you're trying to run this (Cloud or local? Which OS? Which notebook service?) since `raiwidgets` doesn't support everything that exists.
